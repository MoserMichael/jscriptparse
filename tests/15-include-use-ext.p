    # include the source of another file with the use directive
    use "tests/testuseext.js"

    consoleLog("now", "we", "are", 6)
    
