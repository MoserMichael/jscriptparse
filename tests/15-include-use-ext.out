now we are 6
